openbb_platform/core/openbb_core/api/router/commands.py

```diff
@@ -167,6 +167,10 @@ def exclude_fields_from_api(key: str, value: Any):
                 elif is_model(getattr(field, "annotation", None)):
                     exclude_fields_from_api(field_name, getattr(value, field_name))
 
+    # Let a non-OBBject object pass through without validation
+    if not isinstance(c_out, OBBject):
```

Comment:
I was reviewing this again and wanted to ask your opinion if we should at least validate that what the command returns is JSON serializable?

Reply:
FastAPI does this already for the API. From the Python interface, serialization is probably a waste of resources and unnecessary.